### PR TITLE
(Feature) Redeem Position

### LINKED ISSUE #78: [UI/UX] - Redeem position
- Manually enter raw positionId
    - Validate Position exist
    - Validate user has position balance
- Manually enter raw conditionId
    - Validate condition exist
    - Validate condition is resolved
    - Validate condition is included on Position conditions set
- Calculate parentCollection
- Show preview
- Submit

**Wireframe: **
https://wireframepro.mockflow.com/view/M6eaccdb4e3776bf4fd576d04b0095a191590507310129#/page/Dd7446cf37c4871d697dcf50beddc51aa

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #78 

Some (very slight) improvements over what @lmcorbalan did with this section.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr136--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr136--conditionaltokensexplorer.review.gnosisdev.com

File: src/components/form/SelectCondition/index.tsx [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
 
-const InfoMessage = styled.p`
-  color: ${(props) => props.theme.colors.lightGrey};
-  font-size: 14px;
-  font-weight: 400;
-  font-style: italic;
-  line-height: 1.4;
-  margin: 0 0 5px 0;
-  text-align: left;
+import { SelectConditionModal } from '../../modals/SelectConditionModal'
```
Comment (lmcorbalan) [inline src/components/form/SelectCondition/index.tsx:5]: Lets use all absolute or all relative paths

File: src/components/form/SelectCondition/index.tsx [hunk 2]
```diff
@@ -53,35 +43,29 @@ export const SelectCondition = () => {
...
-      </Row>
+      <TitleValue
+        title={title}
+        titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
+        value={
+          <>
+            <Textfield
+              error={!!errors.length}
+              onChange={imputHandler}
+              placeholder={loading ? 'Loading...' : 'Please select a condition...'}
```
Comment (lmcorbalan) [inline src/components/form/SelectCondition/index.tsx:54]: I'd thought about this but the problem I found is when the user manually enters a condition id. 
I think the component should show some hint to the user to let it know data is being fetched/validated so a placeholder wont work in this case. Right?

File: src/components/redeemPosition/SelectPosition/index.tsx [hunk 3]
```diff
@@ -64,15 +49,10 @@ export const SelectPosition = () => {
           <Textfield
             disabled={true}
             error={!!errors.length}
-            placeholder="Please select a position..."
+            placeholder={loading ? 'Loading...' : 'Please select a position...'}
```
Comment (lmcorbalan) [inline src/components/redeemPosition/SelectPosition/index.tsx:52]: Similar scenario with the `Loading...` placeholder in Select Condition component, although this component will be replaced with a `SelectPositionModal` component

File: src/components/form/SelectCondition/index.tsx (hunk 1 above)
Comment (gabitoesmiapodo) [reply src/components/form/SelectCondition/index.tsx:5]: Yeah. We have a linter rule for sorting and grouping imports and such. Absolute paths are messing with that.

File: src/components/form/SelectCondition/index.tsx (hunk 2 above)
Comment (gabitoesmiapodo) [reply src/components/form/SelectCondition/index.tsx:54]: Good point. I think we should improve this through some kind of new design.

I'll create an issue for that, and then implement it.

Comment (gabitoesmiapodo) [reply src/components/form/SelectCondition/index.tsx:54]: Here's the issue https://github.com/gnosis/conditional-tokens-explorer/issues/139

File: src/components/redeemPosition/SelectPosition/index.tsx (hunk 3 above)
Comment (gabitoesmiapodo) [reply src/components/redeemPosition/SelectPosition/index.tsx:52]: Yeah, we'll fix it with https://github.com/gnosis/conditional-tokens-explorer/issues/139

File: src/components/form/SelectCondition/index.tsx [hunk 4]
```diff
@@ -53,35 +43,29 @@ export const SelectCondition = () => {
...
-        />
-      </Row>
+      <TitleValue
+        title={title}
+        titleControl={<TitleControl onClick={openModal}>Select Condition</TitleControl>}
+        value={
+          <>
+            <Textfield
+              error={!!errors.length}
+              onChange={imputHandler}
```
Comment (mariano-aguero) [inline src/components/form/SelectCondition/index.tsx:53]: Don't worry about this wording, is already fixed in another PR

Comment (gabitoesmiapodo) [reply src/components/form/SelectCondition/index.tsx:53]: I don't understand. What should (or shouldn't) I do about this?

**Pull request merged**